packages/analytics-js-integrations/src/integrations/Mixpanel/browser.js

```diff
@@ -98,11 +98,11 @@ class Mixpanel {
 
   isLoaded() {
     window.mixpanel.register({ mp_lib: 'Rudderstack: web' });
-    return !!window?.mixpanel?.config;
+    return !!window?.mixpanel?.__loaded;
   }
 
   isReady() {
```

Comment:
If there is no change in the logic, let's simply invoke `isLoaded` here.

Reply:
Done